error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJMapRules.java

```diff
@@ -148,12 +102,14 @@ MapAssert<K, V> after(Map<K, V> map) {
 
   static final class AbstractMapAssertContainsExactlyInAnyOrderEntriesOf<K, V> {
     @BeforeTemplate
-    AbstractMapAssert<?, ?, K, V> before(AbstractMapAssert<?, ?, K, V> mapAssert, Map<K, V> map) {
+    AbstractMapAssert<?, ?, K, V> before(
+        AbstractMapAssert<?, ?, K, V> mapAssert, Map<? extends K, ? extends V> map) {
```

Comment:
Here and below: I generalized the matching a bit, just because I noticed. This likely applies to _many_ other rules; eventually we'll write an Error Prone check for this.